doc/build-unix.md

```diff
@@ -67,15 +67,18 @@ Build requirements:
 
     sudo apt-get install build-essential libtool autotools-dev automake pkg-config libssl-dev libevent-dev bsdmainutils
 
-On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
+
+Options when installing required Boost library files:
+
+1. On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
```

Comment:
I am not sure but this may be Debian 8+ as implied here: 197a7a8c7ace9ff54010adb836b4e7e35b8ead18